app/konnect/updates.md

```diff
@@ -37,6 +37,11 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 : The Gateway Control Plane Overview page is now updated to present the users all relevant information of a specific control plane. In addition, prompts that nudge users to create their first service, route, plugin, and consumer are presented on the Overview Pages to nudge users to immediately dive into Konnect for the first time.
 
+**{{site.mesh_product_name}} in {{site.konnect_short_name}} (Enterprise and new Plus customers)**
```

Comment:
```suggestion
**{{site.mesh_product_name}} in {{site.konnect_short_name}}**
```

Reply:
I think this needs to be explicit and point to the pricing page. 
As in:
* Enterprise will be able to access mesh immediately and should contact their sales representative. 
* New customers can apply their trial credits towards mesh zones 
For more information view [pricing](pricing). 
Mesh is not available for existing plus customers, stay tuned for more information. 

I would clear this messaging with someone else but explicit is better than not.